index.html

```diff
@@ -33,22 +33,20 @@
         <script src="dist/itowns.js"></script>
         <script src="dist/debug.js"></script>
         <script type="text/javascript">
-            /* global itowns,document,GuiTools*/
+            /* global itowns,document,GuiTools, debug */
 
             var positionOnGlobe = { longitude: 2.351323, latitude: 48.856712, altitude: 25000000 };
 
             // iTowns namespace defined here
             var viewerDiv = document.getElementById('viewerDiv');
             var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe);
             var menuGlobe = new GuiTools('menuDiv', globeView);
-            menuGlobe.view = globeView;
```

Comment:
Good idea. Can you apply this small change to other examples as well?

Reply:
done